Permissions to show reports and tasks test is failing often

The `Contact details page > Permissions to show reports and tasks > should show contact summary that has the full context for reports > 50 validate that the pregnancy card is always displayed` test is failing often.

eg: https://github.com/medic/cht-core/actions/runs/9318549238/job/25652920148#step:16:826

```
 ------------------------------------------------------------------
[chrome-headless-shell 125.0.6422.112 linux #0-3] Running: chrome-headless-shell (v125.0.6422.112) on linux
[chrome-headless-shell 125.0.6422.112 linux #0-3] Session ID: c6be42548c106f784a5bd8bfb0d585ae
[chrome-headless-shell 125.0.6422.112 linux #0-3]
[chrome-headless-shell 125.0.6422.112 linux #0-3] » /tests/e2e/default/contacts/contact-details.wdio-spec.js
[chrome-headless-shell 125.0.6422.112 linux #0-3] Contact details page
[chrome-headless-shell 125.0.6422.112 linux #0-3]     Permissions to show reports and tasks
[chrome-headless-shell 125.0.6422.112 linux #0-3]        ✓ should show reports and tasks when permissions are enabled
[chrome-headless-shell 125.0.6422.112 linux #0-3]        ✖ should show contact summary that has the full context for reports > 50 validate that the pregnancy card is always displayed (5 retries)
[chrome-headless-shell 125.0.6422.112 linux #0-3]        ✓ should not show reports when permission is disabled
[chrome-headless-shell 125.0.6422.112 linux #0-3]        ✓ should not show tasks when permission is disabled
[chrome-headless-shell 125.0.6422.112 linux #0-3]
[chrome-headless-shell 125.0.6422.112 linux #0-3] 3 passing (23.4s)
[chrome-headless-shell 125.0.6422.112 linux #0-3] 1 failing
[chrome-headless-shell 125.0.6422.112 linux #0-3]
[chrome-headless-shell 125.0.6422.112 linux #0-3] 1) Permissions to show reports and tasks should show contact summary that has the full context for reports > 50 validate that the pregnancy card is always displayed
[chrome-headless-shell 125.0.6422.112 linux #0-3] expected '13' to equal '12'      
      actual expected
      
      '1312'
      
[chrome-headless-shell 125.0.6422.112 linux #0-3] AssertionError: expected '13' to equal '12'
[chrome-headless-shell 125.0.6422.112 linux #0-3]     at Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/contact-details.wdio-spec.js:99:52)
------------------------------------------------------------------
```

--- a/tests/factories/cht/reports/pregnancy.js
+++ b/tests/factories/cht/reports/pregnancy.js
@@ -18,7 +18,8 @@ const defaultSubmitter = {
 };
 
 const nextANCVisit = moment().add(2, 'day');
-const lmp = moment().subtract(3, 'months');
+const fixedPregDate = moment('2024-06-03');
+const lmp = fixedPregDate.subtract(84, 'days');
 
 const defaultFields = {
   'inputs': {

